apps/emqx/src/emqx_authentication.erl

```diff
@@ -628,7 +628,6 @@ handle_create_authenticator(Chain, Config, Providers) ->
 do_authenticate([], _) ->
     {stop, {error, not_authorized}};
 do_authenticate([#authenticator{id = ID, provider = Provider, state = State} | More], Credential) ->
-    emqx_metrics_worker:inc(authn_metrics, ID, matched),
```

Comment:
I suggest to record another counter here, i.e. `total`.
And the `rate` is the rate of the `total` counter.

Reply:
It's ok for me